src/Files.App/Views/Settings/GeneralPage.xaml

```diff
@@ -140,10 +140,10 @@
 								</Button.Flyout>
 								<StackPanel Orientation="Horizontal" Spacing="8">
 									<FontIcon
-										Margin="0,1,0,0"
-										FontSize="16"
```

Comment:
General recommendation is **always** use sizes in a multiple of 4. Please use 16.